README.md

```diff
@@ -1 +1,16 @@
-# docker-jenkins
\ No newline at end of file
+# docker-jenkins
+
+This builds a jenkins image and pushes up to quay.
+
+To run jenkins:
+
+1. Request a namespace in the ACP-CI cluster.
+2. Request a keycloak client to be created in the hod-ci realm, the ACP team will create the jenkins secret, which will contain the keycloak json in the necessary namespace.
+3. Create a repo called kube-<your project>-jenkins
+4. `cp -pr {kube,scripts,.drone.yml.example} ../kube-<your project>-jenkins/.`
```

Comment:
There is no drone.yml.example file. Also the secrets being used won't work as they won't be set on the repository once they create their own. So they would need to define the ones you're using for that repository.

Reply:
Er ignore some of that, just saw the drone example yaml in the commit. So the only thing is the secrets bit and the other bits are kube secrets, which is fine, but maybe document that they need to be there or changed etc.